IdokladSdk.IntegrationTests/Tests/Clients/DocumentPayment/SalesDocumentPaymentTests.cs

```diff
@@ -128,6 +128,34 @@ public async Task GetListAsync_ApplyFilter_ReturnsSalesDocumentPayments()
             Assert.That(item.PaidDocument.Id, Is.EqualTo(documentId));
         }
 
+        [Test]
+        public async Task GetListAsync_ApplyDescSort_ReturnsSortedSalesDocumentPayments()
+        {
+            // Act
+            var data = await DocumentPaymentClient.Sales
+                .List().Sort(s => s.DateOfPayment.Desc())
+                .GetAsync()
+                .AssertResult();
+
+            // Assert
+            Assert.That(data.Items, Is.Not.Null);
+            Assert.That(data.Items, Is.Ordered.Descending.By("DateOfPayment"));
```

Comment:
...Descending.By(nameof(SalesDocumentPaymentListGetModel.DateOfPayment)));

Reply:
opraveno. Nunit to nějak nezvádal, když to bylo zanořené. Třeba SalesDocumentPaymentListGetModel.PaidDocument.Id.

Takže jsem ty konkrétní property vyselektoval do jedoduchého pole a tam už assert funguje